skglm/experimental/sqrt_lasso.py

```diff
@@ -132,7 +136,10 @@ def fit(self, X, y):
             Fitted estimator.
         """
         self.coef_ = self.path(X, y, alphas=[self.alpha])[1][0]
-        self.intercept_ = 0.  # TODO handle fit_intercept
+        if self.fit_intercept:
+            self.intercept_ = self.coef_[-1]
```

Comment:
@PascalCarrivain you also need to make self.coef_ shorter by one element in that case, otherwise it still contains the itnercept as last value